src/styles/todoapp.scss

```diff
@@ -131,3 +131,50 @@
     }
   }
 }
+
+.item-enter {
+  max-height: 0;
+}
+
+.item-enter-active {
+  overflow: hidden;
+  max-height: 58px;
+  transition: max-height 0.3s ease-in-out;
+}
+
+.item-exit {
+  max-height: 58px;
+}
+
+.item-exit-active {
```

Comment:
Class names should follow the naming convention with double underscores for nested elements and double dashes for modifiers. Consider renaming '.item-exit-active' to '.item--exit-active'.